test/__snapshots__/DatePicker.test.tsx.snap

```diff
@@ -236,12 +247,6 @@ Object {
               </legend>
             </fieldset>
           </div>
-          <p
-            class="MuiFormHelperText-root Mui-error MuiFormHelperText-sizeMedium MuiFormHelperText-contained Mui-required css-1wc848c-MuiFormHelperText-root"
-            id="mui-9-helper-text"
-          >
-            date must be a \`date\` type, but the final value was: \`Invalid Date\`.
-          </p>
```

Comment:
I'd question the test on this one.

Reply:
Make sure you are looking at latest commit - last commit should have reverted to showing the error correctly

Reply:
entirely possible i messed that up. =)